common/dbc.py

```diff
@@ -31,18 +34,17 @@ def __init__(self, fn):
     self.msgs = {}
 
     self.bits = []
-    for i in range(0, 64, 8):
-      for j in range(7, -1, -1):
+    for i in xrange(0, 64, 8):
+      for j in xrange(7, -1, -1):
         self.bits.append(i+j)
 
     for l in self.txt:
-      l = l.strip()
 
       if l.startswith("BO_ "):
         # new group
         dat = bo_regexp.match(l)
 
-        if dat is None:
+        if not dat:
```

Comment:
In this instance at least, a regex.match call returns either None or a MatchObject. In the case of the MatchObject it would evaluate to True and None should evaluate to False no?


I had assumed that calling __bool__ would be some percent faster for the interpreter than a test for identity.